e3/vcs/svn.py

```diff
@@ -46,6 +46,54 @@ def __init__(self, working_copy):
         """
         self.working_copy = working_copy
 
+    @classmethod
+    def local_url(self, repo_path):
+        """Return the url of a svn repository hosted locally.
+
+        :param repo_path: path to the repo
+        :type repo_path: str
+        :return: the url that can be used as repository url
+        :rtype: str
+        """
+        repo_path = os.path.abspath(repo_path)
+        if sys.platform == 'win32':  # unix: no cover
+            if len(repo_path) > 1 and repo_path[1] == ':':
+                # svn info returns the URL with an uppercase letter drive
+                repo_path = repo_path[0].upper() + repo_path[1:]
+            return 'file:///' + repo_path.replace('\\', '/')
+        else:  # windows: no cover
+            return 'file://' + repo_path
+
+    @classmethod
+    def create(cls, repo_path, initial_content_path=None):
+        """Create a local subversion repository.
+
+        :param repo_path: a local directory that contains the repository
```

Comment:
The wording suggests, at least to me, that repo_path is a directory that already contains the SVN repo. But reading the code, it's actually the location where the repostory will be created, right?

So perhaps just change the documentation to say: "A local directory where to create the repository" ?